### DISCUSSION TITLE
How to correctly specify supplier/manufacture  in metadata files?

### DISCUSSION THREAD

**Discussion by ChristoWolf** [CycloneDX/cyclonedx-dotnet#897] [Q&A] [answered]
Hi!

I am generating SBOMs including a metadata file using `--import-metadata-path` and following [the schema](https://cyclonedx.org/docs/1.2/json/#metadata).

Everything works well except for `supplier` and `manufacture` though.

When specifying
```xml
<manufacture name="some name" url="https://www.company.com" />
...
<supplier name="some name" url="https://www.company.com" />
```
they come out as
```xml
<manufacture />
...
<supplier />
```

I guess it could be related to me not specifying `contact`s in them, but that's an array, so optional from my understanding, right?

Any help would be greatly appreciated :)

**Comment by ChristoWolf** [comment]
Would appreciate any help!

**Comment by mtsfoni** [comment, marked as answer]
Try them as children nodes not as attributes.

**Reply by ChristoWolf** [reply]
Works, thanks for clarifying!
